manifests/integrations/http_check.pp

```diff
@@ -170,6 +174,7 @@
   $password  = undef,
   $timeout   = 1,
   $method    = 'get',
+  $min_collection_interval = 15,
```

Comment:
```suggestion
  $min_collection_interval = undef,
```
We can set it to `undef` here, which will leave it unset (and thus use the default anyway).